extensions/guacamole-auth-ldap/doc/examples/exampleConfigGroup.ldif

```diff
@@ -9,3 +9,4 @@ guacConfigParameter: port=5900
 guacConfigParameter: password=secret
 member: cn=user1,dc=example,dc=com
 member: cn=user2,dc=example,dc=com
+seeAlso: ou=admins,ou=groups,dc=example,dc=com
```

Comment:
JT, you are correct, the documentation example should be seeAlso: cn=admins,ou=groups,dc=example,dc=com @mike-jumper should I open a new issue to fix this?